package.json

```diff
@@ -35,8 +36,11 @@
     "postcss": "^8.2.1",
     "serve": "^11.3.2",
     "simple-git": "^2.24.0",
+    "testcafe": "^1.14.0",
+    "testcafe-browser-provider-browserstack": "^1.13.2-alpha.1",
     "underscore.string": "^3.3.5",
-    "urijs": "1.18.12"
+    "urijs": "1.18.12",
+    "yargs": "^16.2.0"
```

Comment:
oh nit but can we try yargs 17? It was released yesterday

Reply:
I was looking at yargs@17 for my jambo acceptance stuff, and they improved a lot of async/promise related things

Reply:
> It was released yesterday

Here at Yext we live on the cutting edge